build_tools/buildkite/presubmit.yml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Note: this runs on "security: submitted" agents, since this pipeline itself is
+# submitted code and it fetches the scripts from the main branch.
+
+steps:
+  # TODO: Is there a better emoji here? Google logo (would have to add to
+  # Buildkite)?
+  - label: ":face_with_monocle: :admission_tickets: :raised_hand: Check CLA"
```

Comment:
Yeah, let's find something other than this set of emojis

![image](https://user-images.githubusercontent.com/4010439/159094912-a206c2de-d492-4062-b942-0191528544fe.png)

(not blocking this PR, we can change them later)

Reply:
Where'd you get that garbage emoji set?

![image](https://user-images.githubusercontent.com/5732088/159095000-e63e5e9b-3a06-4775-b7b3-ee1910ab2009.png)

Reply:
:( yeah, it looks better on macOS and Android. Windows 10 looks bad here.

https://emojipedia.org/admission-tickets/
https://emojipedia.org/microsoft/windows-10-anniversary-update/admission-tickets/